### PR TITLE
Chart rendering

### PR DESCRIPTION
**Pull request by spireken**

### CODE REVIEW

File: Applications/Spire/Source/Charting/ChartView.cpp [hunk 1]
```diff
@@ -13,6 +13,8 @@ using namespace Nexus;
 using namespace Spire;
 
 namespace {
+  const auto GAP_DIVISOR = 2.95;
```
Comment (kranar) [inline Applications/Spire/Source/Charting/ChartView.cpp]: Make this local to the function using it.

Comment (spireken) [reply Applications/Spire/Source/Charting/ChartView.cpp]: Is there no advantage to declaring it in an anonymous namespace?

Comment (kranar) [reply Applications/Spire/Source/Charting/ChartView.cpp]: Use the most local scope as possible. If something is declared in an anonymous namespace, from my point of view it's a variable I have to be aware of and keep track of for the entire file. If it's local to a function, then I only need to be aware of it for that specific function, so on so forth.

File: Applications/Spire/Source/Charting/ChartView.cpp [hunk 2]
```diff
@@ -278,16 +276,17 @@ void ChartView::paintEvent(QPaintEvent* event) {
       candlestick.GetOpen()});
     auto close = convert_chart_to_pixels({candlestick.GetEnd(),
       candlestick.GetClose()});
+    auto gap_divisor = 2.95;
```
Comment (kranar) [inline Applications/Spire/Source/Charting/ChartView.cpp]: If this is a constant then declare it as such and use CAMEL_CASE.

```
  const auto GAP_DIVISOR = 2.95;
```

**Pull request merged**